'install' doesn't permit install something twice
--- libgda-4.0.2/libgda/Makefile.am.orig	2009-10-28 04:26:53.888863402 +0100
+++ libgda-4.0.2/libgda/Makefile.am	2009-10-28 04:27:03.888176172 +0100
@@ -57,7 +57,6 @@
 	gda-holder.h \
 	gda-lockable.h \
 	gda-log.h \
-	gda-marshal.h \
 	gda-meta-store.h \
 	gda-meta-struct.h \
 	gda-mutex.h \
--- libgda-4.0.2/providers/sqlite/Makefile.am.orig	2009-10-28 04:56:04.268563791 +0100
+++ libgda-4.0.2/providers/sqlite/Makefile.am	2009-10-28 05:08:46.434681971 +0100
@@ -23,7 +23,6 @@
 xmldir   = $(datadir)/libgda-4.0
 xml_in_files = sqlite_specs_create_db.xml.in \
 	sqlite_specs_dsn.xml.in \
-	sqlite_specs_create_db.xml.in \
 	sqlite_specs_drop_db.xml.in \
 	sqlite_specs_create_table.xml.in \
 	sqlite_specs_drop_table.xml.in \
