diff -urN libgda-4.2.0/configure.ac libgda-4.2.0.new//configure.ac
--- libgda-4.2.0/configure.ac	2010-09-26 20:52:03.000000000 +0200
+++ libgda-4.2.0.new//configure.ac	2010-10-01 18:30:53.182115395 +0200
@@ -627,7 +627,7 @@
 	else
 		AC_DEFINE(HAVE_ODBC, 1, [Have ODBC])
 		ODBC_CFLAGS="-I${odbcdir}/include"
-		ODBC_LIBS="-L${odbcdir}/lib ${odbclib}"
+		ODBC_LIBS="${odbclib}"
 	fi
 fi
 
@@ -861,7 +861,7 @@
 		AC_MSG_WARN(tds backend not used)
 	else
 		AC_MSG_CHECKING(for freetds version)
-		freetdsvercheck="`grep \"^#define \(VERSION_NO \|TDS_VERSION_NO \)\" $freetds_incdir/tdsver.h|sed 's|^.*".*v\(.*\)"|\1|'`"
+		freetdsvercheck=$(grep \"^#define \(VERSION_NO \|TDS_VERSION_NO \)\" $freetds_incdir/tdsver.h|sed 's|^.*".*v\(.*\)"|\1|')
 		if test "x$freetdsvercheck" = "x"
 		then
 			AC_MSG_WARN(freetds version check failed - backend not used)
@@ -870,7 +870,7 @@
 		else
 			AC_MSG_RESULT($freetdsvercheck)
 			FREETDS_CFLAGS="-I${freetds_incdir}"
-			FREETDS_LIBS="-L${freetdsdir}/lib -ltds"
+			FREETDS_LIBS="-ltds"
 			AC_DEFINE(HAVE_FREETDS, 1, [Have freetds])
 
 			freetds_version_major="`echo $freetdsvercheck|sed 's:^\([[^.]]*\).*:\1:'`"
@@ -1002,13 +1002,11 @@
 	else
 		SYBASE_DIR=${sybasedir}
 		SYBASE_CFLAGS="-I${sybasedir}/include"
-		SYBASE_LIBS="-lnsl -ldl -lm"
 		AC_MSG_CHECKING(for Sybase host system type)
 		AC_MSG_RESULT($host)
 		case "$host" in
-		*linux*) sybase_ctlibs="-L${sybasedir}/lib -lct -lcs -lsybtcl -lcomn -lintl"
-			sybase_blklibs="-L${sybasedir}/lib -lblk ${sybase_ctlibs}"
-			SYBASE_LIBS="${SYBASE_LIBS} ${sybase_blklibs}"
+		*linux*)
+			SYBASE_LIBS="-lct"
 			AC_CHECK_LIB(ct,tdsdump_on,[
 			AC_MSG_WARN(linking against freetds libct)
 			AC_DEFINE(HAVE_SYBASE, 1, [Have Sybase])
@@ -1229,7 +1227,7 @@
 		AC_MSG_WARN(LDAP backend not used)
 	else
 		LDAP_CFLAGS="-I${ldapdir}/include"
-		LDAP_LIBS="-L${ldapdir}/lib -lldap -llber"
+		LDAP_LIBS="-lldap -llber"
 		AC_DEFINE(HAVE_LDAP, 1, [Have LDAP])
 	fi
 fi
diff -urN libgda-4.2.0/m4/mdbtools.m4 libgda-4.2.0.new//m4/mdbtools.m4
--- libgda-4.2.0/m4/mdbtools.m4	2010-09-24 22:07:47.000000000 +0200
+++ libgda-4.2.0.new//m4/mdbtools.m4	2010-10-01 18:32:05.105428163 +0200
@@ -118,7 +118,7 @@
 	    for d in $mdb_test_dir
 	    do
 	        AC_MSG_CHECKING([for MDB Tools files in $d])
-	        if test -f $d/include/mdbtools.h -a -f $d/$mdb_loclibdir/libmdb$mdb_libext -o -f $d/include/mdbtools.h -a -f $d/$mdb_loclibdir/libmdb.a
+	        if test -f $d/include/mdb/mdbtools.h -a -f $d/$mdb_loclibdir/libmdb$mdb_libext -o -f $d/include/mdb/mdbtools.h -a -f $d/$mdb_loclibdir/libmdb.a
 	        then
   	            save_CFLAGS="$CFLAGS"
 	            CFLAGS="$CFLAGS -I$d/include $mdb_glib_cflags"
@@ -137,8 +137,8 @@
 	            if test x$mdbdir != x
 		    then
 		        AC_MSG_RESULT([found])
-			MDB_CFLAGS=-I${mdbdir}/include
-	    		MDB_LIBS="-L${mdbdir}/lib -lmdb"
+			MDB_CFLAGS=-I${mdbdir}/include/mdb
+	    		MDB_LIBS="-lmdb"
 		        break
   		    else
 		        AC_MSG_RESULT([not found])
